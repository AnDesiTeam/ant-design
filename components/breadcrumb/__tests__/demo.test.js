import demoTest from '../../../tests/shared/demoTest';

demoTest('breadcrumb', { skip: ['react-router.md'], skipTrigger: ['big-data.md'] });
