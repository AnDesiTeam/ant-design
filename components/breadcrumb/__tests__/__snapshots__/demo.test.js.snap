// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/breadcrumb/demo/basic.md correctly 1`] = `
<div
  class="ant-breadcrumb"
>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      Home
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      <a
        href=""
      >
        Application Center
      </a>
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      <a
        href=""
      >
        Application List
      </a>
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      An Application
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
</div>
`;

exports[`renders ./components/breadcrumb/demo/overlay.md correctly 1`] = `
<div
  class="ant-breadcrumb"
>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      Ant Design
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      <a
        href=""
      >
        Component
      </a>
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-overlay-link ant-dropdown-trigger"
    >
      <span
        class="ant-breadcrumb-link"
      >
        <a
          href=""
        >
          General
        </a>
      </span>
      <span
        aria-label="down"
        class="anticon anticon-down"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="down"
          fill="currentColor"
          focusable="false"
          height="1em"
          style="vertical-align:-0.125em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
          />
        </svg>
      </span>
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      Button
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
</div>
`;

exports[`renders ./components/breadcrumb/demo/router.md correctly 1`] = `
<div
  class="demo"
>
  <div
    class="demo-nav"
  >
    <a
      href="#/"
    >
      Home
    </a>
    <a
      href="#/apps"
    >
      Application List
    </a>
  </div>
  Home Page
  <div
    class="ant-alert ant-alert-info ant-alert-no-icon"
    data-show="true"
    style="margin:16px 0;"
  >
    <span
      class="ant-alert-message"
    >
      Click the navigation above to switch:
    </span>
    <span
      class="ant-alert-description"
    />
  </div>
  <div
    class="ant-breadcrumb"
  >
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <span>
          Home
        </span>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
  </div>
</div>
`;

exports[`renders ./components/breadcrumb/demo/router-4.md correctly 1`] = `
<div
  class="demo"
>
  <div
    class="demo-nav"
  >
    <a
      href="#/"
    >
      Home
    </a>
    <a
      href="#/apps"
    >
      Application List
    </a>
  </div>
  Home Page
  <div
    class="ant-alert ant-alert-info ant-alert-no-icon"
    data-show="true"
    style="margin:16px 0;"
  >
    <span
      class="ant-alert-message"
    >
      Click the navigation above to switch:
    </span>
    <span
      class="ant-alert-description"
    />
  </div>
  <div
    class="ant-breadcrumb"
  />
</div>
`;

exports[`renders ./components/breadcrumb/demo/separator.md correctly 1`] = `
<div
  class="ant-breadcrumb"
>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      Home
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      &gt;
    </span>
  </span>
  <span>
    <a
      class="ant-breadcrumb-link"
      href=""
    >
      Application Center
    </a>
    <span
      class="ant-breadcrumb-separator"
    >
      &gt;
    </span>
  </span>
  <span>
    <a
      class="ant-breadcrumb-link"
      href=""
    >
      Application List
    </a>
    <span
      class="ant-breadcrumb-separator"
    >
      &gt;
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      An Application
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      &gt;
    </span>
  </span>
</div>
`;

exports[`renders ./components/breadcrumb/demo/withIcon.md correctly 1`] = `
<div
  class="ant-breadcrumb"
>
  <span>
    <a
      class="ant-breadcrumb-link"
      href=""
    >
      <span
        class="anticon"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          fill="currentColor"
          focusable="false"
          height="1em"
          width="1em"
        >
          <use
            href="#icon-home-o"
          />
        </svg>
      </span>
    </a>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <a
      class="ant-breadcrumb-link"
      href=""
    >
      <span
        class="anticon"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          fill="currentColor"
          focusable="false"
          height="1em"
          width="1em"
        >
          <use
            href="#icon-user-o"
          />
        </svg>
      </span>
      <span>
        Application List
      </span>
    </a>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <span
      class="ant-breadcrumb-link"
    >
      Application
    </span>
    <span
      class="ant-breadcrumb-separator"
    >
      /
    </span>
  </span>
</div>
`;
